test/src/test/java/org/corfudb/runtime/collections/CorfuStoreShimTest.java

```diff
@@ -248,7 +249,7 @@ public void checkRevisionValidation() throws Exception {
         }
         assertThat(entry.getMetadata().getRevision()).isEqualTo(2L);
         assertThat(entry.getMetadata().getCreateUser()).isEqualTo("user_1");
-        assertThat(entry.getMetadata().getLastModifiedTime()).isLessThan(System.currentTimeMillis());
+        assertThat(entry.getMetadata().getLastModifiedTime()).isLessThan(System.currentTimeMillis() + 1);
```

Comment:
Please add a comment on why you are comparing with a future timestamp. I believe you did it to avoid having to sleep() which I think is a great idea, but would be good to document.

Reply:
Yeah, probably then @zfrenette 's idea of using `isLessThanOrEqualTo` would be self-describing, now it seems that we compare with the future